package-lock.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "docs",
+  "name": "rollkit-docs",
```

Comment:
I think it is because the `package.json` file in the Rollkit docs does not have a project name, that's why it automatically changed to the repo name of ours (since we already have a `docs` repo in `artela-network`, so we renamed the forked repo to `rollkit-docs`).